pkg/client/cache/listers.go

```diff
@@ -380,3 +380,42 @@ func (s *StoreToJobLister) GetPodJobs(pod *api.Pod) (jobs []extensions.Job, err
 	}
 	return
 }
+
+// Typed wrapper around a store of PersistentVolumes
+type StoreToPVFetcher struct {
+	Store
+}
+
+// GetPersistentVolumeInfo returns cached data for the PersistentVolume 'id'.
+func (s *StoreToPVFetcher) GetPersistentVolumeInfo(id string) (*api.PersistentVolume, error) {
+	o, exists, err := s.Get(&api.PersistentVolume{ObjectMeta: api.ObjectMeta{Name: id}})
+
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving PersistentVolume '%v' from cache: %v", id, err)
+	}
+
+	if !exists {
+		return nil, fmt.Errorf("PersistentVolume '%v' is not in cache", id)
```

Comment:
I agree; but this is effectively a copy-and-paste of GetNodeInfo (also in pkg/client/cache/listers.go).  I'm happy to change this to return either (*api.PersistentVolume, bool, error) or (*api.PersistentVolume, error), so not-found would be (_, false, nil) or (nil, nil).  What do you think?